cfgrammar/src/lib/yacc/ast.rs

```diff
@@ -228,6 +229,79 @@ impl GrammarAST {
         }
         Ok(())
     }
+
+    /// Return an iterator over all the `YaccGrammarWarning` produced by this grammar.
+    pub fn warnings(&self) -> impl Iterator<Item = YaccGrammarWarning> + '_ {
```

Comment:
I think we can get rid of this `'_` lifetime

Reply:
Should be fixed in [1de1fc0](https://github.com/softdevteam/grmtools/pull/345/commits/1de1fc06e43373e771c3cde9f8a246a25f3dfe11)